brianlong/www.validators.app
# 183981415-remove-home-new

#### What's this PR do?
- remove unused home_new path with corresponding code

#### How should this be manually tested?
- run tests
- see if you can find anything else to delete

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/183981415)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
